Project/Engine/src/sysFrame/DirectX/DirectXCommon.cpp

```
@@ -508,6 +509,53 @@ void DirectXCommon::UpdateFixFPS() {
     reference_ = std::chrono::steady_clock::now();
 }
 
+#include <Psapi.h>
+void DirectXCommon::DisplayInfo() {
```

Cppcheck (2.10-2)

[style] The function 'DisplayInfo' is never used.

(unusedFunction)